manifest_staging/charts/secrets-store-csi-driver/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: secrets-store-csi-driver
-version: 1.2.2
-appVersion: 1.2.2
+version: 1.3.2
+appVersion: 1.3.2
```

Comment:
Can you revert this change? We will update the version while releasing the chart